packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -31,8 +31,8 @@ function BlockToolbar( { blockClientIds, isValid, mode } ) {
 			{ mode === 'visual' && isValid && (
 				<>
 					<BlockSwitcher clientIds={ blockClientIds } />
-					<BlockControls.Slot />
-					<BlockFormatControls.Slot />
+					<BlockControls.Slot bubblesVirtually className="editor-block-toolbar__slot" />
```

Comment:
Unfortunate 😕 A bigger task than we can solve here, but might be worth revisiting #7231 so we can consolidate the behavior and avoid the wrapper that makes this restyling necessary.

Reply:
If it's just to apply the one rule below, could we perhaps set it inline? I think inline styles can be good when we want it to be "invisible" rather than the class to be used for "real" styling.

Reply:
Ah, I misread this as being applied to *all* slots. So never mind my comment. :)

Reply:
I don't understand why this change is necessary though, without any comment.

Reply:
It's just a style fix. Without it the toolbar is broken (appears on multiple lines)

Reply:
Well, more specifically, the style fix is caused by the introduction of a new `div` wrapper which hadn't existed previously. It's a bit worrisome, since it further enforces the inconsistency between `bubblesVirtually={ true }` and `bubblesVirtually={ false }` (there is no `div` wrapper if `bubblesVirtually={ false }`, and `className` has no effect).

Do you see something like #7231 being possible to implement? I recall there being some issues last time it was explored, but I think avoiding the `div` altogether helps a number of difficulties.